Detectors/MUON/MID/Raw/src/LocalBoardRO.cxx

```diff
@@ -0,0 +1,42 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MID/Raw/src/LocalBoardRO.cxx
+/// \brief  Structure to store the FEE local board information
+/// \author Diego Stocco <Diego.Stocco at cern.ch>
+/// \date   19 November 2019
+
+#include "MIDRaw/LocalBoardRO.h"
+
+#include <iostream>
+#include <sstream>
+#include <MIDRaw/CrateParameters.h>
+
+namespace o2
+{
+namespace mid
+{
+std::ostream& operator<<(std::ostream& os, const LocalBoardRO& loc)
+{
+  /// Stream operator for LocalBoardRO
+
+  // Using stringstream here so that we can write part of the message in hexadecimal
+  // without modifying the full stream
+  std::stringstream ss;
```

Comment:
Note that you might consider using fmt::format for those kind of things.

Reply:
I was actually searching for something similar, thanks!
Implemented in the new version